src/AspNet.Security.OAuth.Automatic/AutomaticAuthenticationDefaults.cs

```diff
@@ -0,0 +1,49 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using Microsoft.AspNetCore.Builder;
+
+namespace AspNet.Security.OAuth.Automatic {
+    /// <summary>
+    /// Default values used by the Automatic authentication middleware.
+    /// </summary>
+    public static class AutomaticAuthenticationDefaults {
+        /// <summary>
+        /// Default value for <see cref="AuthenticationOptions.AuthenticationScheme"/>.
+        /// </summary>
+        public const string AuthenticationScheme = "AutomaticLabs";
```

Comment:
I supposed `Automatic` was not a valid scheme because it conflicts with the special value used by the aspnet/Security stack?